aboutpage.html

```diff
@@ -0,0 +1,174 @@
+         <li class="active"><a id=""  href="index.html">Home</a></li>
+          <li><a href="aboutpage.html">About</a></li>
+         <li><a class="nav-list-content" href="index.html#tech-topics">Tech topics</a></li>
+         <li><a class="nav-list-content" href="index.html#contributors">Top Contributors</a></li>
+       </ul>
+      </nav>
+     </header>
+
+      <!-- mobile nav list ends here -->
+
+      <!-- about section starts here -->
+
+  <main>
+    <section id="about-section">
+      <div class="container">
+        <div class="about-hero-container">
+          <div>
+            <div class="hero-headline-title">  
+              <h1 class="hero-headline-title text-accent-400 ">READ MORE ABOUT DIV <br>NEWS RIGHT HERE </h1>
+            </div>
+            <div class="about-text">
+            <p>Div News is rated the number one news source for all things tech related,<br> Founded by our top contributors in 1995 we stand as the longest running news source for the tech industry.
+              We are proud of the ever growing numbers. as well as our rapidly growing contributors. Have something tech related to share? find out how you too can become a top contributor, by contributing informstion via our news lead intake system, simply send an email with the tech related topic you'd like to contribute to
+            </p>
+            </div>
+            <p>please contact us by Email for any further questions or to join our contributors
+            </p>
+            <div class="email-container">
+            <a class="email" href="mailto:name@email.com">contributor@divnews.com</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+      <!-- about section ends here -->
+
+      <!-- collaborations section starts here -->
+
+    <section id="collaborations">
+      <div class="container">
+        <div>
+          <h2 class="title fs-700 text-neutral-900 fw-bold">Lets collaborate</h2>
+          <div class="indicator"></div>
+        </div>
+        <div class="collaborations-container">
+          <div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corrupti vitae labore expedita?<br> Aliquam consectetur assumenda laborum ullam accusantium. Aliquid voluptatum in consequatur. Delectus maxime labore,<br> numquam voluptate fuga soluta aliquam.</p>
+          </div>
+          <div >
+            <img  class="divnewsaboutlogo" src="img/divnewsaboutlogo.png" alt="divnewsaboutlogo">
+          </div>
+        </div>
+      </div>
+    </section>
+
+      <!-- collaborations section ends here -->
+    
+      <!-- past collaborators section starts here -->
+
+    <section>
+      <div class="container">
+        <div class="past-collaborators">
+          <h2 class="title fs-700 text-neutral-900 fw-bold">These are our past collaborators</h2>
+          <div class="indicator"></div>
+          <p>Join our wall of collaborators</p>
+        </div>
```

Comment:
[OPTIONAL] I think you did a great job here however there is an issue. Kindly make sure that your `h2` text is not overlapping your image above in your "about" page desktop version. Kindly fix this. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/198833192-349da0d3-65ad-468c-8ec2-b3ddb73f2e21.png)